assets/js/components/ExecutionResults/CheckResultDetail/ExpectationsResults.stories.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { faker } from '@faker-js/faker';
+import ExpectationsResults from './ExpectationsResults';
+
+export default {
+  title: 'ExpectationsResults',
+  component: ExpectationsResults,
+  args: {
+    isTargetHost: true,
+    results: [
+      { name: faker.animal.bear(), return_value: true },
+      { name: faker.animal.bear(), return_value: false },
+    ],
+    isError: false,
+    errorMessage: 'An error occurred',
+  },
+};
+
+export function Default(args) {
```

Comment:
Fow new stories we're aiming to follow the way of writing them that has been introduced in the latest versions of storybook https://storybook.js.org/docs/react/writing-stories/introduction

See [Accordion](https://github.com/trento-project/web/blob/main/assets/js/components/Accordion/Accordion.stories.jsx) and [CheckResultDetail](https://github.com/trento-project/web/blob/main/assets/js/components/ExecutionResults/CheckResultDetail/CheckResultDetail.stories.jsx) and https://github.com/trento-project/web/pull/1428#discussion_r1200129544

Reply:
Oh thanks for reminding me!  :+1: